src/chain-spec.ts

```diff
@@ -433,6 +467,19 @@ export function writeChainSpec(specPath: string, chainSpec: any) {
   }
 }
 
+// helper
+function getRandom(arr: string[], n: number) {
```

Comment:
I think it is better for helpers to exist under `utils`?

Reply:
Yes, we can move to `utils` since is very generic.

Reply:
done, thanks!!